src/directLine.ts

```diff
@@ -308,7 +308,8 @@ export class DirectLine implements IBotConnection {
     private streamUrl: string;
     public referenceGrammarId: string;
 
-    private pollingInterval: number = 1000;
+    private pollingInterval: number = 1000; //ms
+    private readonly POLLING_INTERVAL_LOWER_BOUND: number = 50; //ms
```

Comment:
Should we move it to `const` outside of this class?

`50` should be `200`.

```suggestion
    private readonly POLLING_INTERVAL_LOWER_BOUND: number = 200; //ms
```

Reply:
why would we move it outside of the class?

Reply:
The primary reason is that its value cannot be changed from the initial assignment when movd outside the class. A static or a class member can be changed even though its defined as readonly.

Reply:
to elaborate, given the following example:
```js
class MyClass {
  private static readonly A_VALUE = 0;
  
  private readonly B_VALUE = 1;
}
```
I can change any of these values simply by using:
```js
MyClass['A_VALUE'] = -1; // change static
new MyClass()['VALUE_B'] = -2; // change class member
```